pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -43,19 +43,10 @@ def __init__(self, trainer: "pl.Trainer") -> None:
     def on_trainer_init(
         self,
         logger: Union[bool, Logger, Iterable[Logger]],
-        flush_logs_every_n_steps: Optional[int],
         log_every_n_steps: int,
         move_metrics_to_cpu: bool,
     ) -> None:
         self.configure_logger(logger)
-        if flush_logs_every_n_steps is not None:
-            rank_zero_deprecation(
-                f"Setting `Trainer(flush_logs_every_n_steps={flush_logs_every_n_steps})` is deprecated in v1.5 "
-                "and will be removed in v1.7. Please configure flushing in the logger instead."
-            )
-        else:
-            flush_logs_every_n_steps = 100  # original default parameter
-        self.trainer.flush_logs_every_n_steps = flush_logs_every_n_steps
```

Comment:
@awaelchli I have currently removed trainer.flush_logs_every_n_steps attribute. Do you think this is suitable?

Reply:
Yes!